src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -637,6 +636,7 @@ $pf-v5-c-dropdown--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg"
 // Loading spinner
 .#{$dropdown}__toggle-progress {
   position: absolute;
+  left: var(--#{$dropdown}__toggle--m-split-button--child--PaddingLeft);
```

Comment:
@mcoker This needs to be consistently aligned with padding, ie doesn't need a unique variable.